generator-plugin/src/main/java/uk/gov/justice/maven/generator/io/files/parser/generator/GenerateMojo.java

```diff
@@ -93,4 +102,27 @@ private GenerateGoalConfig configuration(final List<Path> sourcePaths) {
 
     }
 
+    public void loadProjectDependencies() throws DependencyResolutionRequiredException, MalformedURLException {
```

Comment:
Can we separate this method into a class then it could be unit tested

Reply:
Done